File: PspDataPresenter.Core/Models/StopwatchModel.cs
using System;

namespace PspDataPresenter.Core
{
    public class StopwatchModel
    {
        public RoomModel Room { get; set; }
        public DateTime Start { get; set; }
        public DateTime End { get; set; }
        // start, stop
        public string EventType { get; set; }
        public int EndMessageDisplayInterval { get; set; }
        public DateTime Timestamp { get; set; }
    }
}

File: PspDataPresenter.App/data-presenter-app/src/app/_core/models/stopwatch.model.ts
import { RoomModel } from './room.model';
export interface StopwatchModel {
    room: RoomModel;
    start: Date | string;
    end: Date | string;
    eventType: string;
    endMessageDisplayInterval: number;
    timestamp: Date | string;
}

export enum StopwatchEventType {
    Start,
    Stop
}